docs/en/latest/FAQ.md

```diff
@@ -70,3 +70,6 @@ $ swagger generate spec -o ./docs/en/latest/api/api.yaml --scan-models
 ```shell
 $ swagger-markdown -i ./docs/en/latest/api/api.yaml
 ```
+### 6. What is the default strategy when import a duplicate route?
```

Comment:
```suggestion

### 6. What is the default strategy when import a duplicate route?
```

Reply:
![image](https://user-images.githubusercontent.com/2106987/115568090-89a2dc00-a2ee-11eb-87cf-e28c4a8f1d24.png)

Seems no change?

Reply:
may be for leaving a line🤔?

Reply:
yep, only to fix the lint error

Reply:
👍👍